test/extensions/INTEL/SPV_INTEL_bindless_images/bindless_images_generic.ll

```diff
@@ -0,0 +1,47 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv --spirv-ext=+SPV_INTEL_bindless_images
+; RUN: llvm-spirv %t.spv -o %t.spt --to-text
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.spv -o %t.rev.bc -r --spirv-target-env=SPV-IR
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck < %t.rev.ll %s --check-prefix=CHECK-LLVM
+
+; RUN: not llvm-spirv %t.bc 2>&1 | FileCheck %s --check-prefix=CHECK-ERROR
+; CHECK-ERROR: RequiresExtension: Feature requires the following SPIR-V extension:
+; CHECK-ERROR-NEXT: SPV_INTEL_bindless_images
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
+target triple = "spir64-unknown-unknown"
+
+; CHECK-SPIRV: Capability BindlessImagesINTEL
+; CHECK-SPIRV: Extension "SPV_INTEL_bindless_images"
+; CHECK-SPIRV-DAG: TypeVoid [[#VoidTy:]]
+; CHECK-SPIRV-DAG: TypeInt [[#Int32Ty:]] 32
+; CHECK-SPIRV-DAG: TypeInt [[#Int64Ty:]] 64
+; CHECK-SPIRV-DAG: Constant [[#Int64Ty]] [[#Const42:]] 42 0
+; CHECK-SPIRV-DAG: TypeImage [[#IntImgTy:]] [[#Int32Ty]]
+; CHECK-SPIRV-DAG: TypeSampler [[#SamplerTy:]]
+; CHECK-SPIRV: FunctionParameter [[#Int32Ty]] [[#Input:]]
+; CHECK-SPIRV: ConvertHandleToImageINTEL [[#IntImgTy]] [[#]] [[#Input]]
+; CHECK-SPIRV: ConvertHandleToSamplerINTEL [[#SamplerTy]] [[#]] [[#Const42]]
```

Comment:
Should we also check addressing model of the input IDs as it's mentioned in spec?